@xen-orchestra/lite/src/locales/en.json

```diff
@@ -47,6 +47,9 @@
   "page-not-found": "This page is not to be found…",
   "password": "Password",
   "password-invalid": "Password invalid",
+  "patch": {
+    "missing": "{n} missing"
+  },
```

Comment:
Should we actually namespace this? I'm not sure: on one hand, it prevents us from reusing it later, but on the other hand, it might be necessary, it might not be the same message depending on what we're talking about (→ 🇫🇷).